subscription-engine/kafka-ws/README.md

```diff
@@ -200,19 +200,20 @@ REDIS_HOST       | Redis host                   | redis
 REDIS_PORT       | Redis port                   | 6379                               | number |
 REDIS_DATABASE   | Redis database               | 1                                  | number |
 
-Note1: Maximum lifetime of a connection will be the highest value between the time calculate from expiration timestamp provided by the jwt token (if KAFKA_WS_JWT_EXP_TIME is true) and KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_MAX_LIFE_TIME is -1, it will be the value provided in the time calculate from expiration timestamp provided by the jwt token (if KAFKA_WS_JWT_EXP_TIME is true), otherwise there would be no maximum connection lifetime.
+Note1: Maximum lifetime of a connection will be the highest value between the time calculate from expiration timestamp provided by the JSON Web Token (JWT)  (if KAFKA_WS_JWT_EXP_TIME is true) and KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_MAX_LIFE_TIME is -1, it will be the value provided in the time calculate from expiration timestamp provided by the JSON Web Token (JWT)  (if KAFKA_WS_JWT_EXP_TIME is true), otherwise there would be no maximum connection lifetime.
```

Comment:
Suggestion:
-------------------
Note1: A websocket connection is closed by the server when certain conditions are met. If  KAFKA_WS_JWT_EXP_TIME is set to true, the server will consider this value for closing the connection if it is greater than the  KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_JWT_EXP_TIME is set to false and KAFKA_WS_MAX_LIFE_TIME is set to -1, the server will never close a connection by its duration.

Reply:
Done